code/modules/clothing/shoes/boots.dm

```diff
@@ -59,8 +59,8 @@
 
 /obj/item/clothing/shoes/jackboots/Initialize(mapload)
 	. = ..()
-
 	create_storage(storage_type = /datum/storage/pockets/shoes)
```

Comment:
Will this not have an affect on it? Matches fit in jackboots
https://github.com/tgstation/tgstation/blob/a46523adb68553a49be819eabce6966bc130eb29/code/datums/storage/subtypes/pockets.dm#L113

Reply:
It resolves before storage

Reply:
Is it still possible to put it in storage though? I don't even know if they have visible inventories or if it's only accessible through clicking on the boot itself to put in and alt click to take out.